src/handlers.jl

```diff
@@ -261,6 +261,11 @@ function is_complete_request(socket, msg)
                            Dict("status"=>status, "indent"=>"")))
 end
 
+function interrupt_request(socket, msg)
+    @async Base.throwto(requests_task[], InterruptException())
```

Comment:
Why is this `@async`?

Reply:
Because it blocks the control port otherwise indefinitely(?), at least that is what I observed. This is parallel to the construction in the main loop.